vignettes/daily-series.Rmd

```diff
@@ -101,6 +91,7 @@ proc_keyword_latest("Rezession", "CH")
 ```
 
 This function downloads raw daily, weekly, and monthly data for the specified pair of keyword and geo location.
+If the data for a particular keyword is not yet available, `proc_keyword_init()` should be called instead.
```

Comment:
the function downloaded time series (d,w,m) for the keyword "Insvolvenz". The function above, proc_keyword_latest I could not execute "could not find function "proc_keyword_latest". I guess this is to come?

Reply:
great! thanks for the explanation and hint!